content/project/projects/bridges/voc/contents.lr

```diff
@@ -25,6 +25,7 @@ implemented, and there is only a bare bones standard library implementation.
 However, it is possible to convert simple Python programs, and even write
 simple Android applications.
 
+To use VOC you need to have `Java 8` and `Apache Ant` installed. Start using VOC with the 'Getting Started'_ guide.
```

Comment:
I like having the link to the "Getting Started" guide, but let's nix the "To use VOC... " sentence.